src/constructor/BlogConstructorProvider.tsx

```diff
@@ -35,6 +38,7 @@ export const BlogConstructorProvider: React.FC<BlogConstructorProviderProps> = (
         <RouterContext.Provider value={router} key="router-context" />,
         <MobileContext.Provider value={Boolean(isMobile)} key="is-mobile-context" />,
         <DeviceContext.Provider value={device} key="device-context" />,
+        <SettingsContext.Provider value={settings} key="settings-context" />,
```

Comment:
add info about this context in context README

Reply:
done